README.md

```diff
@@ -102,13 +102,9 @@ All changes or deprecations of connector features will be announced in the [rele
 * Client libraries
   * [Go][go-oidc]
 
-## Reporting a security vulnerability
+## Reporting a vulnerability
```

Comment:
We also need to update [the index page](https://github.com/dexidp/website/blob/main/content/docs/_index.md#reporting-a-security-vulnerability) on the website.